docs/en/stack/monitoring/how-monitoring-works.asciidoc

```diff
@@ -2,32 +2,37 @@
 [[how-monitoring-works]]
 == How monitoring works
 
-Monitoring collects data from {es} nodes, Logstash nodes, and {kib} instances.
-The {es} cluster you are monitoring controls where the monitoring metrics for
-the entire stack are stored. By default, they are stored in local indices. In
-production, we strongly recommend using a separate monitoring cluster. Using a
-separate monitoring cluster prevents production cluster outages from impacting
+Monitoring collects data from {es} nodes, Logstash nodes, and {kib} instances. 
+
+In general, the {es} cluster you are monitoring controls where the monitoring 
+metrics for the stack are stored. By default, they are stored in local indices. 
+
+In production, we strongly recommend using a separate monitoring cluster. Using 
+a separate monitoring cluster prevents production cluster outages from impacting
 your ability to access your monitoring data. It also prevents monitoring
-activities from impacting the performance of your production cluster.
+activities from impacting the performance of your production cluster. 
+
+beta[] In 6.5 and later, you can use {metricbeat} to collect and ship data about 
```

Comment:
I like that we are flagging this as a "beta" section, including the diagram that follows it. What do you think about also keeping the original non-beta way of doing things, including the diagram? Do you think it might be too much information and could lead to confusion?